### ISSUE TITLE
"ArrayGet: array type required" when calling nnedi3resize()

### ISSUE THREAD

**Issue by Kogarou-san** [Dogway/Avisynth-Scripts#80]
**Describe the bug**
```
ArrayGet: array type required.
(C:/Program Files (x86)/AviSynth+/plugins64+/TransformsPack - Main.avsi, line 448)
(C:/Program Files (x86)/AviSynth+/plugins64+/ResizersPack.avsi, line 825)
(C:/Program Files (x86)/AviSynth+/plugins64+/ResizersPack.avsi, line 827)
(C:/Program Files (x86)/AviSynth+/plugins64+/ResizersPack.avsi, line 365)
```

**To Reproduce**
```cpp
ColorBarsHD() 
nnedi3resize(720,540)
```
Happened with at least YUV444P8 and YUV420P16 formats. Frame props as created by ColorBarsHD() and on a DVD with the frame props as:
```cpp
Frame: 107891
Keys: 11
_ChromaLocation (0[left])
_DurationDen (60000)
_DurationNum (1001)
_FieldBased (0[progressive])
_FieldOrder (1)
_Matrix (2[undef])
_PictType (B)
_Primaries (2[undef])
_SARDen (11)
_SARNum (10)
_Transfer (2[undef])
```

**Expected behavior**
To not crash

**Files**
None

**System:**
 - Windows Server 2022 21H2
 - AviSynth+ 3.7.3 (r4003, 3.7, x86_64)
 - Script pack commit https://github.com/Dogway/Avisynth-Scripts/commit/b52c94c2de439fa6694fc3865946e50f86880d77

**Additional context**

**Comment by Dogway** [comment]
I'm not able to reproduce your issue...
I tried with ColorBarsHD() in both YV24 and YUV444P8, and on SD and 720p.
Also revised the lines in the error and didn't see an issue.
Try to see if everything is updated or share a snippet source.

Your commit link is for "Stabilization Tools Pack", is this related?

**Comment by Kogarou-san** [comment]
>Your commit link is for "Stabilization Tools Pack", is this related?
I took a long time off of encoding and started updating every script and DLL a few days ago. It just happened that *that* was  the latest commit, which I downloaded.
Since you couldn't reproduce the issue I took a proper look into the autoload folder and I found the culprit: I failed to delete 2 old versions of TransformsPack that I probably had in an old backup.
I am aware such things can happen and I used to run Avisynth Info Tool, but I forgot to download and run it at the time.
I am very sorry to have wasted your time with an issue caused by me.
